### DISCUSSION TITLE
How to change different background color to splash screen in dark mode???

### DISCUSSION THREAD

**Discussion by saimarshadsaim31** [obytes/react-native-template-obytes#407] [Q&A]
I am trying to change the background color of my splash screen to white when my phone is in dark mode and it is not changing I have tried everything but nothing works. I have tried this:
```
splash: {
    image: './assets/icon-test.png',
    resizeMode: 'contain',
    backgroundColor: '#ffffff',
  }```

//and also tried to do it using plugins in app.config:
[
      'expo-splash-screen',
      {
        image: './assets/icon.png',
        resizeMode: 'contain',
        backgroundColor: '#ffffff',
      },
 ]
//and also like this:
[
      'expo-splash-screen',
      {
        image: './assets/icon.png',
        resizeMode: 'contain',
        backgroundColor: '#ffffff',
        dark: {
          image: './assets/icon.png',
          backgroundColor: '#ffffff',
        },
      },
 ]
```
on light mode the background works fine, I am using eas and have build both the development build and apk but still it shows black background in dark mode

**Comment by yjose** [comment]
As they mention in the [docs](https://docs.expo.dev/versions/latest/sdk/splash-screen/#configuration) you are doing it right.

You may need to create an issue in the expo repo as this feature is still new

**Reply by saimarshadsaim31** [reply]
Thanks for the reply @yjose. Ok, I will post it in the expo repo. I posted it here first because it is only happening like this in the obytes template if I create a brand new project and give different colors to the splash screen background it picks up that color.
